src/lib.rs

```diff
@@ -158,6 +158,16 @@ pub fn main() -> Result<(), JsValue>
             let elem = document.get_element_by_id("canvas").expect("canvas handle");
             elem.dyn_into::<HtmlCanvasElement>()?
         };
+
+    {
+        let canvas_clone = canvas.clone();
+        let callback = move |event: web_sys::FocusEvent|
```

Comment:
You can rename `event` to `_event` to suppress the compiler warning about unused variables

Reply:
3d25492